apps/Gruntfile.js

```diff
@@ -93,7 +93,7 @@ describe('entry tests', () => {
     'poetry',
     'studio',
     'turtle',
-    'weblab'
+    'weblab',
```

Comment:
This is the diff that will be resolved by https://github.com/code-dot-org/code-dot-org/pull/51565 right? (Also thanks for doing that!)